src/build.gradle

```diff
@@ -3,7 +3,9 @@ buildscript {
 
     repositories {
         mavenCentral()
-        jcenter()
+        maven { // required for detekt < v1.17.0
+            url "https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven"
```

Comment:
Doesn't this need a version number ?

Reply:
Good question. No, because it's a repository not a package.

And the package version is defined implicitly by dependency tree of detekt (v1.14).